packages/components/src/components/floating-menu/floating-menu.js

```diff
@@ -325,21 +325,32 @@ class FloatingMenu extends mixin(
         const primaryFocusNode = this.element.querySelector(
           this.options.selectorPrimaryFocus
         );
-        const focusableNode = this.element.querySelector(
+
+        const contentNode = this.options.contentNode || this.element;
+        const tabbableNode = contentNode.querySelector(
           settings.selectorTabbable
         );
 
+        // The programmatically focusable element may be (and typically will be) the content node itself;
+        const focusableNode = contentNode.matches(settings.selectorFocusable)
+          ? contentNode
+          : contentNode.querySelector(settings.selectorFocusable);
+
         if (primaryFocusNode) {
+          // User defined focusable node
           primaryFocusNode.focus();
+        } else if (tabbableNode) {
+          // First sequentially focusable node
+          tabbableNode.focus();
         } else if (focusableNode) {
+          // First programmatic focusable node
           focusableNode.focus();
         } else {
           this.element.focus();
           if (__DEV__) {
-            const elementTabindex = this.element.getAttribute('tabindex');
             warning(
-              elementTabindex !== null && parseInt(elementTabindex, 10) > -1,
-              'Floating Menus without interactive elements must include tabindex="0" on the floating element.'
+              focusableNode === null,
+              'Floating Menus must at least a programmatic focusable child by adding tabindex="-1" to the content element.'
```

Comment:
can we reword this to be a bit clearer?

Reply:
@emyarod good catch... sometimes my brain goes faster than my fingers.  I've updated the comment.